content/03-types/02-relational/07-what-are-joins.mdx

```diff
@@ -0,0 +1,401 @@
+
+In one-to-one relations, JOINs are straightforward. You can use a traditional `INNER JOIN` to combine the users with their profiles. Here is an example query:
+
+```sql
+SELECT
+		user.id, user.email, profile.name, profile.gender, profile.age
+FROM
+		user
+INNER JOIN profile
+		ON user.email = profile.user_email;
+```
+
+When performing an `INNER JOIN` between the two tables, we would get a result set where each row contains columns from both tables, matched based on the relationship criteria. Here is an example result set:
+
+| id  | email             | name    | gender | age |
+| --- | ----------------- | ------- | ------ | --- |
+| 1   | meera@prisma.io   | Meera   | Female | 30  |
+| 2   | xander@prisma.io  | Xander  | Male   | 30  |
+| 3   | zara@prisma.io    | Zara    | Female | 32  |
+| 4   | artemis@prisma.io | Artemis | Male   | 32  |
+
+When you join the `User` and `Profile` tables using the `INNER JOIN` and the specified join condition, each row in the result set represents a user along with their corresponding profile. In other words, we include the `id` and `email` columns from the user table, along with the `name`, `gender`, and `age` of those users from the `Profile` table.
+
+### One-to-many (1-n)
+
+In a one-to-many relationship, each record in one table can be associated with multiple records in another table. Here is an example SQL schema demonstrating a one-to-many relation between `User` and `Post` models:
+
+```sql
+CREATE TABLE "User" (
+    "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    "email" TEXT NOT NULL
+);
+
+CREATE TABLE "Post" (
+    "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    "title" TEXT NOT NULL,
+    "authorId" INTEGER NOT NULL,
+    CONSTRAINT "Post_authorId_fkey" FOREIGN KEY ("authorId") REFERENCES "User" ("id") ON DELETE RESTRICT ON UPDATE CASCADE
+);
+
+CREATE UNIQUE INDEX "User_email_key" ON "User"("email");
+
+```
+
+This means that one user can have multiple posts, but each post is linked to only one user.
+
+When dealing with one-to-many relations, traditional JOINs can lead to duplicated data. Consider the following query:
+
+```sql
+SELECT
+		user.id, user.email, post.id AS post_id, post.title AS post_title
+FROM
+		user
+INNER JOIN post
+		ON user.id = post.authorId;
+
+```
+
+Here is an example result set based on the query above:
+
+| id  | email           | post_id | post_title                 |
+| --- | --------------- | ------- | -------------------------- |
+| 1   | jenny@prisma.io | 1       | Introduction to Prisma ORM |
+| 1   | jenny@prisma.io | 2       | Introduction to JOINs      |
+| 1   | jenny@prisma.io | 3       | Introduction to SQL        |
+| 2   | meera@prisma.io | 4       | Using Prisma Client        |
+
+As you can see in the result set, Jenny's email gets duplicated over multiple rows for each of her posts. As we incorporate more nested relations, result sets may become bloated. This can make it harder to interpret the data.
```

Comment:
not only Jenny's `email` but also the `id`

Reply:
🙏  [7ab8e61](https://github.com/prisma/dataguide/pull/470/commits/7ab8e61191519b32075674502b3253276d101d2b)